### ISSUE TITLE
Some doubts about the method

### ISSUE THREAD

**Issue by QiaoranC** [rhfeiyang/PPOT#1]
Hi, I have some doubts about the method.
- How sensitive is the proposed method to the assumption of a uniform prior distribution in the KL divergence constraint for pseudo-label generation?
- Can you provide additional insights or empirical evidence supporting the choice of gradually increasing the fraction of high-confident samples (ρ) and its impact on model performance?
- Could you elaborate on the specific efficiency gains achieved by the proposed solver in solving the unbalanced optimal transport problem?
- Why are experiments not conducted on ImageNet datasets, especially when discussing pre-trained models? What considerations led to the choice of datasets, and are there potential limitations introduced by this selection?
- In the context of imbalanced datasets, have you considered discussing the limitations of clustering accuracy (ACC), normalized mutual information (NMI), and F1-score as evaluation metrics? Are there other metrics or analyses that could provide a more comprehensive evaluation?
- How sensitive is the proposed method's performance to the choice of hyperparameters such as λ, ϵ, and initial ρ? Have you conducted a sensitivity analysis or ablation study to explore these aspects?

**Comment by kleinzcy** [comment]
Hi @QiaoranC , thanks for your interest. 

* As we do not know data distribution, we naturally assume a uniform prior distribution. The superiority results on various datasets with different imbalance ratios indicate our method is not sensitive to the uniform prior distribution assumption. 
* Please refer to Figure 4 and Table 3 for empirical evidence.
*  We have not fully understood the reason and are still working on it. Currently, we speculate it is correlated with the tightness of inequality constraint.
* As unsupervised learning is secretly clustering the data, and DINO is pre-trained on balanced ImageNet, we do not experiment on ImageNet. Our datasets are selected from existing and popular benchmarks and we do not select datasets based on results. To some extent, the results of those datasets show the generalization of our method.
* The evaluation of clustering, especially in imbalanced scenarios is still a challenging problem. We follow previous work to adopt the ACC, NMI, and F1-score for evaluation.
* We set $ \lambda,\epislon $ as the default value for all experiments and analysis $ \rho $ in the experiments.

At last, we have to say that if your comment is generated by the chat model, it would be better if you could adjust the chat model to be more gentle.
